main.go

```diff
@@ -131,6 +134,7 @@ func Router(Server *socketio.Server) *mux.Router {
 	r.HandleFunc("/users/{user_id}", auth.IsAuthenticated(user.DeleteUser)).Methods("DELETE")
 	r.HandleFunc("/users", auth.IsAuthenticated(user.GetUsers)).Methods("GET")
 	r.HandleFunc("/users/{email}/organizations", auth.IsAuthenticated(user.GetUserOrganizations)).Methods("GET")
+	r.HandleFunc("/users/invite", user.CreateUserFromUUID).Methods("POST")
```

Comment:
This invite, are we inviting people to join zuri chat and also handling inviting people to join an org on zuri chat?

Reply:
This is built in response to the invite that guests receive to join an organization on zurichat. It serves a second purpose of adding and verifying new users to zurichat.

Reply:
Okay, great.